routers/web/repo/issue.go

```diff
@@ -518,6 +518,12 @@ func RetrieveRepoMilestonesAndAssignees(ctx *context.Context, repo *repo_model.R
 }
 
 func retrieveProjects(ctx *context.Context, repo *repo_model.Repository) {
+	// Distinguish whether the owner of the repository
+	// is an individual or an organization
+	repoOwnerType := project_model.TypeIndividual
+	if repo.Owner.IsOrganization() {
```

Comment:
Can you ensure repo.Owner has been loaded from database? Maybe invoke `repo.LoadOwner` first.

Reply:
I checked before calling `retrieveProjects` that the owner has been loaded from the database. Maybe we need to load it again in `retrieveProjects` to ensure safe?